platforms/core-execution/worker-processes/build.gradle.kts

```diff
@@ -15,6 +15,7 @@ dependencies {
     api(project(":messaging"))
     api(project(":problems-api"))
     api(project(":process-services"))
+    api("com.google.code.findbugs:jsr305:3.0.2")
```

Comment:
This should be `libs.jsr305`

Reply:
https://github.com/gradle/gradle/pull/28087